### ISSUE TITLE
Fastconform SSL pretrained recipe failed potentially due to PTL 2.0

### ISSUE THREAD

**Issue by XuesongYang** [NVIDIA/NeMo#7507]
**Describe the bug**

the `examples/asr/speech_pretraining/speech_pre_training.py` is not working properly on the lates main branch, but working in 1.20. At first glance, the error could be related to the upgrade of Pytorch lightning 2.0.

**Error Brief**
```bash
lightning_fabric.utilities.exceptions.MisconfigurationException: `ModelCheckpoint(monitor='val_loss')` could not find the monitored key in the returned metrics: ['learning_rate', 'global_step', 'train_contrastive', 'train_backward_timing in s', 'train_step_timing in s', 'epoch', 'step']. HINT: Did you call `log('val_loss', value)` in the `LightningModule`?
```

**Error Details**

[error_logs.zip](https://github.com/NVIDIA/NeMo/files/12719689/error_logs.zip)

**Comment by XuesongYang** [comment]
It seems this PR will fix the error: https://github.com/NVIDIA/NeMo/pull/7505

**Comment by titu1994** [comment]
Good catch, for some reason PR was closed though.. @KunalDhawan ?

**Comment by KunalDhawan** [comment]
I've opened a new PR for the issue to keep a consistent format of logging across RNNT, CTC, and Hybrid models - https://github.com/NVIDIA/NeMo/pull/7531. Once this is approved we can make the same changes for SSL, SLU and other models

**Comment by nithinraok** [comment]
@XuesongYang are you still seeing these errors?

**Comment by XuesongYang** [comment]
I switched back to r1.20.0 release, and it worked well. I haven't tried it on the latest r1.21.0 yet. Do we have a nightly test for the minimalist fastconformer SSL training recipe?

**Comment by nithinraok** [comment]
We have SSL training notebook, and it was fixed for PTL upgrade.

**Comment by XuesongYang** [comment]
Let's close this issue then. Thanks!
